packages/amplify_api_plugin_interface/lib/amplify_api_graphql_helpers_interface.dart

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
uber nit: might as well update the year from 2020 to 2021 in these comments for all these files.

Reply:
No worries, updated.